tests/Feature/IDealResponseTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Event;
+use App\User;
+use App\Participant;
+use App\Facades\Mollie;
+use Tests\TestCase;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Mail;
+
+class FakePaymentMetadata
+{
+    public $type = "existing";
+    public $camp_id;
+    public $participant_id;
+
+    function __construct(Event $evt, Participant $part)
+    {
+        $this->camp_id = $evt->id;
+        $this->participant_id = $part->id;
+    }
+}
+class FakeEventPayment
+{
+    public $metadata;
+    function __construct(Event $evt, Participant $part)
+    {
+        $this->metadata = new FakePaymentMetadata($evt, $part);
+    }
+
+    function isPaid()
+    {
+        return true;
+    }
+}
+
+
+
+class IDealResponse extends TestCase
+{
+
+    public $event;
+    public $user;
+
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function testExample()
+    {
+        $response = $this->get('/');
```

Comment:
Deze basic test lijkt me hier niet op zijn plaats (binnen deze class). In mijn eigen branch heb ik deze al opgenomen in een aparte test case, dus wat mij betreft mag deze hier worden verwijderd.

Reply:
Oeps, stukje wat ik vergeten ben weg te halen uit de standaard test structure die artisan aanmaakte.